recipe/meta.yaml

```diff
@@ -28,20 +21,22 @@ build:
 requirements:
   build:
     - python      # [win]
+    - readline 6.2*
     - vc 9  # [win and py27]
     - vc 10  # [win and py34]
     - vc 14  # [win and (py35 or py36)]
   run:
+    - readline 6.2*
```

Comment:
@msarahan we are still pinning to `sqlite 3.13.*` but, b/c `sqlite` is used to to build python and we may change that pinning in the future, I would like your opinion on the use of `readline` here.